Presentations/06-R-programming-practices.Rmd

```diff
@@ -809,68 +735,92 @@ name: looping
 
 # Looping
 
-#### `sapply(X, FUN, ...)`: applies a function to all elements of a vector or list and returns the result in a vector. Its arguments are:
+## Apply
```

Comment:
Since we are using the apply family, I think we would start by adding there a 3 diff: apply, sapply, lapply, and how thy differ. We could introduce the map family from purrr since we used the tidyverse.

Reply:
that would be a nice addition, but I'm afraid I don't have enough time to add it before Wednesday. I'll make sure to include this in an issue with the ideas for the next edition